### ISSUE TITLE
SendableChoosers not shown on SmartDashboard if Driver Station run first

### ISSUE THREAD

**Issue by KenBaron** [wpilibsuite/SmartDashboard#70]
If the FRC Driver Station 17.0a11 is executed, which then automatically executes the java SmartDashboard (2017.2.1) the SmartDashboard does not display the SendableChoosers. If the SmartDashboard is run first, the SendableChoosers work. I'm not sure if the same problem would occur with the prior version.

**Comment by KenBaron** [comment]
When this is tried on a Windows 8 laptop, the problem did not occur, but it continues to occur on my Windows 10 laptop. The laptops are set up the same way except that the Windows 10 laptop has the Vision processing components installed from option 2 (non Labview team) on the kickoff DVD. Note that we have about 8 SendableChoosers that are String types.

**Comment by AustinShalit** [comment]
I have opened a fix for this issue (#72).  I can send you a link to download and test if you want.

**Comment by KenBaron** [comment]
Sure, thanks. In case you did not see it I wrote an issue against (https://github.com/wpilibsuite/allwpilib/issues/471) that should probably have been written against SmartDashboard.

**Comment by AustinShalit** [comment]
Try this (unzip & run):

[SmartDashboard-all.jar.zip](https://github.com/wpilibsuite/SmartDashboard/files/743174/SmartDashboard-all.jar.zip)

**Comment by KenBaron** [comment]
Thanks for the jar, but I see no change in behavior. I renamed your jar and placed it in the location of the SmartDashboard.jar file. Almost every time (works about 10% of the time) I run the Driver Station followed by the SmartDashboard, the SendableChoosers do not show up. Yet when I run the SmartDashboard first, the SendableChoosers always show up.

But yesterday, with a different laptop, they always showed up. So, I looked deeper.

When I run SmartDashboard.jar I see in Task Manager that it is run under C:\Program Files\Java\jre1-8-0_121CameWith_JDK\bin\javaw.exe

But when the SmartDashboard is executed via starting the Driver Station it is run under C:\ProgramData\Oracle\Java\javapath_target_16324656\java.exe

I'm not sure why or if that is the cause of the issue I am having.
